src/content/docs/pl/concepts/why-astro.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Dlaczego Astro?
+description: "Astro to wszechstronny framework do budowania szybkich stron internetowych skoncentrowanych na dostarczaniu treści. Dowiedz się więcej."
+i18nReady: true
+---
+
+Astro to **wszechstronny** **framework** do budowania **szybkich** stron internetowych skoncentrowanych na **dostarczaniu treści**.
+
+Dlaczego wybrać Astro, a nie inny webowy framework? Oto pięć głównych projektowych zasad, które pomogą wytłumaczyć dlaczego zbudowaliśmy Astro, problemy, które rozwiązuje, i dlaczego Astro może być najlepszym wyborem dla Twojego projektu lub zespołu:
+
+#### Astro jest...
+
+1. [Skoncentrowany na treści](#skoncentrowany-na-treści): Astro jest zaprojektowane dla stron bogatych w treść.
+2. [Server-first](#server-first): Strony działają szybciej, kiedy HTML jest generowany na serwerze.
+3. [Domyślnie szybki](#domyślnie-szybki): Zbudowanie wolnej strony za pomocą Astro powinno być niemożliwe.
+4. [Łatwy w użyciu](#łatwy-w-użyciu): Nie musisz być ekspertem, żeby budować z Astro.
+5. [Pełny funkcji, ale elastyczny](#pełny-funkcji-ale-elastyczny): Ponad 100 integracji z Astro do wyboru.
+
+## Skoncentrowany na treści
+
+**Astro zostało zaprojektowane do budowania stron bogatych w treść.** Należą do nich większość stron marketingowych, wydawniczych, dokumentacje, blogi, portfolia oraz niektóre strony e-commerce.
+
+W przeciwieństwie do tego podejścia większość współczesnych webowych frameworków jest zaprojektowana w celu budowania *aplikacji internetowych*. Te frameworki sprawdzają się najlepiej przy tworzeniu bardziej zaawansowanych, przypominających aplikacje, doświadczeń: panele admina, skrzynki odbiorcze, serwisy społecznościowe, listy zadań, a nawet aplikacje podobne do natywnych jak [Figma](https://figma.com/) lub [Ping](https://ping.gg/).
+
+Jest to jedna z najważniejszych różnic, która pomaga zrozumieć Astro. Unikalny dla Astro nacisk na treść pozwala nam na podejmowanie decyzji, które pomagają dostarczyć niezrównaną wydajność oraz funkcjonalności, których nie miałoby sensu implementować we frameworkach służących do tworzenia aplikacji.
+
+:::tip
+Jeśli Twój projekt wpisuje się w kategorię "aplikacji", Astro może nie być właściwym wyborem... **i jest to w porządku!** Sprawdź [Next.js](https://next.org/), alternatywny do Astro framework do budowania aplikacji webowych.
+:::
+
+
+## Server-first
```

Comment:
```suggestion
## Serwer w pierwszej kolejności
```

Wydaje mi się że powinniśmy również tłumaczyć nagłówki

Reply:
myślałem na początku żeby tłumaczyć całą tą frazę, ale nie znalazłem żadnego naturalnego odpowiednika, i zasugerowałem się innymi tłumaczeniami, gdzie "server-first" jest zostawione nieprzetłumaczone. Wydaje mi się to w porządku, szczególnie że potem jest poświęcona cała sekcja, która tłumaczy o co chodzi